grr/server/grr_response_server/output_plugins/elasticsearch_plugin.py

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/env python
+# Lint as: python3
```

Comment:
No need for these lines (no one needs nor should run it as a stand-alone executable and nowadays everything is linted in Python 3 mode anyway).